query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/InsertingNullInRequiredFieldsSpec.scala

```diff
@@ -67,8 +67,8 @@ class InsertingNullInRequiredFieldsSpec extends FlatSpec with Matchers with ApiS
         |  }
         |}""",
       project,
-      errorCode = 2012,
-      errorContains = """Missing a required value at `Mutation.createA.data.ACreateInput.key"""
+      errorCode = 2009,
+      errorContains = """Mutation.createA.data.ACreateInput.key`: A value is required but not set."""
```

Comment:
same like the other comment (just marking every occurence to easily spot all the changed behavior for others)